main.go

```diff
@@ -172,7 +172,7 @@ func main() {
 		setupLog.Error(err, "unable to start app mesh controller")
 		os.Exit(1)
 	}
-	cloud, err := aws.NewCloud(awsCloudConfig, metrics.Registry)
+	cloud, err := aws.NewCloud(awsCloudConfig, metrics.Registry, injectConfig.UseAwsDualStackEndpoint, injectConfig.UseAwsFIPSEndpoint)
```

Comment:
Can the flags be placed on `aws.CloudConfig` instead of `inject.Config`? That would be closer to existing conventions, and eliminate the need to add two unstructured arguments here.

Reply:
I replaced flags from `inject.Config` to `aws.CloudConfig`.